docs/testrunner-toolkit/configuration.md

```diff
@@ -178,6 +179,207 @@ suites:
       testFiles: [ "**/*.*" ] # Cypress native glob support.
 ```
 
+## Running in CI
+
+Please visit [our CI integrations page](/testrunner-toolkit/integrations) for more information about how to run your tests in the following CI platforms:
+
+* [Jenkins](/testrunner-toolkit/integrations/jenkins)
+* [GitHub Actions](/testrunner-toolkit/integrations/github-actions)
+
+## Common Syntax Reference
```

Comment:
@dpgraham I think this section moved to a new page. Might want to pull and check the latest refs.

- [Common Syntax](https://docs.saucelabs.com/testrunner-toolkit/configuration/common-syntax)
- [New config page](https://docs.saucelabs.com/testrunner-toolkit/configuration)